src/iosrv/init.c

```diff
@@ -262,8 +276,11 @@ server_init()
 	/* initialize the network layer */
 	if (sys_map_path != NULL)
 		flags |= CRT_FLAG_BIT_PMIX_DISABLE;
+	ctx_nr = dss_sys_xs_nr + dss_tgt_nr;
+	if (dss_tgt_offload_xs_nr >= 1)
+		ctx_nr += dss_tgt_nr;
```

Comment:
ctx_nr += dss_tgt_nr * dss_tgt_offload_xs_nr ? then you may not need if at all?

Reply:
one problem is only the first offload XS will create cart context for now.